sdk/communication/ci.yml

```diff
@@ -38,3 +38,5 @@ extends:
         safeName: azurecommunicationphonenumbers
       - name: azure-communication-network-traversal
         safeName: azurecommunicationnetworktraversal
+      - name: azure-communication-short-codes
```

Comment:
this should be azure-rest-communication-short-codes and azurerestcommunicationshortcodes since this package is in scope @azure-tools